src/core_plugins/testbed/public/testbed.html

```diff
@@ -5,6 +5,10 @@
     you need to demonstate your functionality. Nothing need be preserved. -->
     <!-- CONTENT START -->
 
+    <div class="kuiControlledTable">
+      <loading-results results-name="all the things"></loading-results>
```

Comment:
You're absolutely right... we should combine these into a single component. Not sure why I made it two in the first place! How about `kuiTableInfo`?